demos/appsflyer-react-native-app/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
   package="com.appsflyerexample">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
+      tools:replace="android:allowBackup"
```

Comment:
What is this? This seems to cause a breaking change @al-af:

```
> Task :app:processInternalReleaseMainManifest FAILED
/Users/swrobel/Code/myapp/android/app/src/main/AndroidManifest.xml:6:161-188 Error:
	Attribute application@allowBackup value=(false) from AndroidManifest.xml:6:161-188
	is also present at [com.appsflyer:af-android-sdk:6.14.0] AndroidManifest.xml:32:9-35 value=(true).
	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:6:2-33:16 to override.
```

Reply:
@swrobel @al-af any update on when this will be fixed? We want to upgrade to use the latest appsflyer sdk that includes the required privacy manifest for iOS but this is a breaking change that prevents android from building.

Similar error I received after upgrading:
```
Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:26:9-36
	is also present at [com.appsflyer:af-android-sdk:6.14.0] AndroidManifest.xml:32:9-35 value=(true).
	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:7:5-117 to override.
```

Reply:
@al-af Could you please let me know when this will be fixed? If we change the **android:allowBackup="true" in AndroidManifest.xml** then it's making an issue with our app features like app auto upgradation from the Play Store.
Thanks.